easybuild/easyblocks/o/openmpi.py

```diff
@@ -113,7 +113,9 @@ def config_opt_used(key, enable_opt=False):
             # for OpenMPI v4.x, the openib BTL should be disabled when UCX is used;
             # this is required to avoid "error initializing an OpenFabrics device" warnings,
             # see also https://www.open-mpi.org/faq/?category=all#ofa-device-error
-            if LooseVersion(self.version) >= LooseVersion('4.0.0') and '--with-ucx' in self.cfg['configopts']:
+            is_ucx_enabled = ('--with-ucx' in self.cfg['configopts'] and
```

Comment:
Since we now enforce "--with-ucx=path" does this really work correctly?

Reply:
Yes. `--with-ucx=path` means enabled as does `--with-ucx`, but `--with-ucx=no` means the opposite

Reply:
I mainly mean, does '--with-ucx' in self.cfg['configopts'] return true if we have --with-ucx=path in configopts?

I can never remember how that "in" operator works...